app/models/profile.rb

```diff
@@ -56,20 +51,12 @@ def canonical?
     parent_profile_id.blank?
   end
 
-  def destroy_orphaned_business_objective
-    bo_changes = (previous_changes.fetch(:business_objective_id, []) +
-                  changes.fetch(:business_objective_id, [])).compact
-    return if bo_changes.blank?
-
-    BusinessObjective.without_profiles.where(id: bo_changes).destroy_all
-  end
-
-  def clone_to(account: nil, host: nil, external: true)
+  def clone_to(account: nil, host: nil, policy: nil)
```

Comment:
Observation: This method is used in XccdfReprtParser, but the policy isn't passed.

Reply:
`nil` policy is the case when `external: true`. So this method is supposed to be used to clone a canonical profile to a user's account, for both external reports and internal.